components/places/src/db/db.rs

```diff
@@ -61,13 +61,18 @@ impl PlacesDb {
             -- files in memory, since on Android there's no tmp partition. See
             -- https://github.com/mozilla/mentat/issues/505. Ideally we'd only
             -- do this on Android, and/or allow caller to configure it.
+            -- (although see also bug 1313021, where Firefox enabled it for both
```

Comment:
That bug does what we're doing, more or less, I think

Reply:
Yeah - it was more to clarify that we don't necessarily only want it on Android.